content/en/contact.md

```diff
@@ -24,6 +24,8 @@ Email: [sponsor@letsencrypt.org](mailto:sponsor@letsencrypt.org)
 
 For information about various technical mailing lists, please visit our [Get Involved](/getinvolved/) page.
 
+To sign up for our newsletter, [click here.] (https://eepurl.com/gi8Ohr)
```

Comment:
```suggestion
To sign up for our newsletter, [click here.] (https://letsencrypt.us14.list-manage.com/subscribe?u=dc00f5bdab7179027a494d4e8&id=ec773c6bad)
```

I think we should avoid using link shorteners for links on our website. This is the expanded version of the link you had here.